tests/unit/grading/test_jplag.py

```diff
@@ -0,0 +1,66 @@
+from tempfile import TemporaryDirectory
+
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.test import TestCase
+
+from inloop.grading.copypasta import jplag_check
+from inloop.solutions.models import SolutionFile
+
+from tests.unit.grading.mixins import PlagiatedSolutionsData
+
+FIBONACCI = """
+public class Fibonacci {
+    public static int fib(final int x) {
+    if (x < 0) {
+        throw new IllegalArgumentException("x must be greater than or equal zero");
+    }
+
+    int a = 0;
+    int b = 1;
+
+    for (int i = 0; i < x; i++) {
+        int sum = a + b;
+        a = b;
+        b = sum;
+    }
+
+    return a;
+    }
+
+    /*
+     * This string is just here for testing purposes.
+     */
+}
+"""
+
+
+class JPlagCheckTest(PlagiatedSolutionsData, TestCase):
```

Comment:
We should think about tagging this test, because it is actually a slower integration test and not a unit test. For example, all tests involving Docker have the `slow` tag.